.zuul.yaml

```diff
@@ -62,49 +132,64 @@
     name: stf-crc-ocp_412-nightly_bundles
     parent: stf-crc-nightly_bundles
     description: |
-      Deploy STF using the nightly bundles on OCP 4.12
-    vars:
-      crc_ocp_bundle: 'https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.12.13/crc_libvirt_4.12.13_amd64.crcbundle'
+      Deploy STF using the nightly bundles on OCP 4.12.13
```

Comment:
I assume that this output now has a patch level because that's what's in the extracted images. If for some reason we wanted a newer 4.12 we'd have to move to CRC 2.19.(z>0) and remember to update these lines too?

Reply:
Yes. I added the version here so we can be clear about what's being used. It's not strictly necessary, but I wanted some indication of what OCP version was being run.
I can also add a check_ocp_version.yml and an `ocp_version` var to the jobs so it is explicitly checked, and we know if the test environment is changed.

In the future, should we require a newer OCP 4.12 (and it's available in a new CRC release), then the nodesets at the top of the file would be updated once the images are available in zuul.

Reply:
Using the x.y.z of OCP version may create an undue burden on the maintainer.